src/components/Pricing/PricingTable/EnterpriseModal.js

```diff
@@ -0,0 +1,106 @@
+                                <dl className="flex justify-between mb-0 p-2">
+                                    <dt className="mb-0 opacity-75 text-xs">First 10 million</dt>
+                                    <dd className="mb-0 font-bold text-xs">$4,500</dd>
+                                </dl>
+                                <dl className="flex justify-between mb-0 p-2">
+                                    <dt className="mb-0 opacity-75 text-xs">10-100 million</dt>
+                                    <dd className="mb-0 font-bold text-xs">$0.00009</dd>
+                                </dl>
+                                <dl className="flex justify-between mb-0 p-2">
+                                    <dt className="mb-0 opacity-75 text-xs">100 million - 1 billion</dt>
+                                    <dd className="mb-0 font-bold text-xs">$0.000018</dd>
+                                </dl>
+                                <dl className="flex justify-between mb-0 p-2 pb-3">
+                                    <dt className="mb-0 opacity-75 text-xs">More than 1 billion</dt>
+                                    <dd className="mb-0 font-bold text-xs">$0.0000036</dd>
+                                </dl>
+                            </div>
+
+                            <div className="flex justify-between items-baseline">
+                                <div className="mb-0 text-sm font-bold text-almost-black text-opacity-75">
+                                    Monthly minimum price
+                                </div>
+                                <div className="mb-0 flex items-baseline">
+                                    <div className="text-base">{monthlyMinimumPrice}</div>
+                                    <div className="opacity-50">/mo</div>
+                                </div>
+                            </div>
+
+                            <hr className="border-gray-accent-light bg-transparent border-dashed border-r-0 border-b-0 border-left-0 my-2 border-t" />
+
+                            <div className="flex justify-between items-baseline">
+                                <div className="text-base mb-0 font-bold">Estimated price</div>
+                                <div className="mb-0 font-bold flex items-baseline">
+                                    <div className="text-base">${finalCost}</div>
+                                    <div className="opacity-50">/mo</div>
+                                </div>
+                            </div>
+                        </Plan>
+                    </div>
+                    <button
+                        className="absolute top-2 right-2 sm:top-3 sm:right-3 lg:top-6 lg:right-6"
+                        onClick={() => setOpen(false)}
+                    >
+                        <Close className="w-3 h-3 sm:w-auto sm:h-auto" />
+                    </button>
+                </div>
+            </div>
+        </Modal>
+    )
+}
```

Comment:
Tested both of the sliders out with negative numbers etc, the behavior looks reasonable.